testplan/runnable/base.py

```diff
@@ -1165,31 +1170,91 @@ def _pre_exporters(self):
                 self._result.test_report
             )
 
-    def _invoke_exporters(self):
+    def _run_exporter(
+        self,
+        exporter: Exporter,
+        source: TestReport,
+        export_context: ExportContext,
+    ) -> ExporterResult:
+        """
+        Wraps an exporter run and handles exceptions.
+
+        :param exporter: exporter to run
+        :param source: Testplan report to export
+        :param export_context: information about other exporters
+        :return: ExporterResult object containing information about the actual exporter object and its possible output
+        """
+
+        traceback_string = None
+        exp_result = None
+        try:
```

Comment:
i think we'd better check the method signature here to make things more explicit

Reply:
There are opposite opinions, discuss this on the team meeting.